app/helpers/application_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
```

Comment:
Not a correction right now since you don't know what you're gonna use or not, but when you start a new project always remember to remove the files you won't need. Also applies to gems in the `gemfile`. For example, if you're going to make a pure API, remove anything having to do with views/forms/visual stuff/jquery/etc